packages/webview_flutter/webview_flutter/example/lib/main.dart

```diff
@@ -292,6 +340,16 @@ class SampleMenu extends StatelessWidget {
       children: cookieWidgets.toList(),
     );
   }
+
+  static Future<String> _prepareLocalFile() async {
+    final String tmpDir = (await getTemporaryDirectory()).path;
+    final File indexFile = File('$tmpDir/www/index.html');
+
+    await Directory('$tmpDir/www').create(recursive: true);
```

Comment:
You could just use `indexFile.create(recursive: true);`

If you don't go that route, you should do something like:
```
final String parentDir = '$tmpDir/www';
final File indexFile = File('parentDir/index.html');
await Directory(parentDir).create(recursive: true);
```

so there's not a duplication of partial paths that could get out of sync.

Reply:
(Also, you should ideally build these as arrays of components and `.join(Platform.pathSeparator)` them so it's correct on Windows, as this package is eventually expected to support all platforms.)

Reply:
Fixed.